components/x-topic-search/src/TopicSearch.jsx

```diff
@@ -43,8 +43,26 @@ const topicSearchActions = withActions(({ minSearchLength = 2, maxSuggestions =
 	}
 }));
 
-const TopicSearch = topicSearchActions(({ searchTerm, showResult, result, actions, isLoading, csrfToken }) => (
-	<div className={ classNames(styles['container']) }>
+const TopicSearch = topicSearchActions(({ searchTerm, showResult, result, actions, isLoading, csrfToken, followedTopics }) => {
+
+	document.body.addEventListener('x-follow-button', event => {
```

Comment:
The `x-follow-button` event just means a button has been clicked on, it doesn't mean the follow/unfollow action has been successfully performed. Also, if a topic is followed via some other means, this won't know about it.

I think it would be better to listen for the events from myft-client, like here: https://github.com/Financial-Times/n-myft-ui/blob/master/components/x-button-integration/index.js#L27